modules/tensor_mechanics/doc/content/source/userobjects/CavityPressureUserObject.md

```diff
@@ -21,6 +21,14 @@ original pressure, temperature, and volume) plus the amount in the cavity due to
 during the simulation.  The volume $V$ is provided by a single postprocessor value or a vector of
 postprocessor values that are summed together before being used in [eq:ideal_gas_law].
 
+There is the option to specify additional volumes with corresponding temperatures are not meshed,
+but communicate directly with the cavity. In this case the pressure becomes:
```

Comment:
There is an option to specify additional, unmeshed volumes with corresponding temperatures that communicate directly with the cavity.